documentation/apis/administration/1.0.0/api.yaml

```diff
@@ -186,13 +186,6 @@ paths:
         Cette requête retourne l'ensemble des services connus par l'administrateur. 
         Cette liste peut être vide si aucun service n'a été configuré.
       operationId: "services"
-      parameters:
-        - name: "withId"
-          description: "Si on souhaite récupérer les ids des services. Ces ids sont normalement dans la configuration de l'administrateur. "
-          in: "query"
-          required: false
-          schema:
-            type: "boolean"
       responses:
```

Comment:
Thanks for the parameter ! 
In the response, can you add a 500 under the 200 ? 
```
500:
          description: "Internal error"
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/errorResponse"
```
That is the format of the error.